src/Form/Extension/AttributeOptionTypeExtension.php

```diff
@@ -28,6 +29,21 @@ public function buildForm(FormBuilderInterface $builder, array $options)
         ]);
         $builder->add('constraints', TextType::class, ['required' => true]);
         $builder->add('type_config', TextType::class, ['required' => true]);
+
+        $transformer = new class implements DataTransformerInterface {
```

Comment:
Can you please use a physical data transformer class instead of an anonymous one?